tools/development/start.sh

```diff
@@ -55,7 +55,7 @@ if [[ ! -z ${1} ]] && [[ ${1} == "--link" ]]; then
             cp -as /mnt/open-space-toolkit-core/include/OpenSpaceToolkit/Core /usr/local/include/OpenSpaceToolkit/Core; \
             cp -as /mnt/open-space-toolkit-core/src/OpenSpaceToolkit/Core/* /usr/local/include/OpenSpaceToolkit/Core/; \
             ln -s /mnt/open-space-toolkit-core/lib/libopen-space-toolkit-core.so /usr/local/lib/; \
-            ln -s /mnt/open-space-toolkit-core/lib/libopen-space-toolkit-core.so.0 /usr/local/lib/; \
+            ln -s /mnt/open-space-toolkit-core/lib/libopen-space-toolkit-core${SHARED_LIBRARY_TARGET}.${PROJECT_VERSION_MAJOR} /usr/local/lib/; \
```

Comment:
```suggestion
            ln -s /mnt/open-space-toolkit-core/lib/${SHARED_LIBRARY_TARGET}.${PROJECT_VERSION_MAJOR} /usr/local/lib/; \
```
Also I am not sure those env vars can be obtained in here 🤔

Reply:
Yeah they might have to be passed... I'll check if it works

Reply:
Okay fixed it, thank you for the thorough review on this.